src/SFA.DAS.Roatp.CourseManagement.Domain/ApiModels/Provider.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace SFA.DAS.Roatp.CourseManagement.Domain.ApiModels
+{
+    public class Provider
+    {
+        public int Ukprn { get; set; }
+        public string LegalName { get; set; }
+        public string TradingName { get; set; }
+        public string Email { get; set; }
+        public string Phone { get; set; }
+        public string Website { get; set; }
+        public string MarketingInfo { get; set; }
+        public decimal? EmployerSatisfaction { get; set; }
+        public decimal? LearnerSatisfaction { get; set; }
```

Comment:
We should not be exposing feedback data out of outer api. This is not a concern for web and should be filtered out. Actually we should filter these out at inner api level.

Reply:
removed